src/widget/form/chatform.cpp

```diff
@@ -1085,5 +1085,33 @@ void ChatForm::hideNetcam()
 
 void ChatForm::retranslateUi()
 {
+    QString callObjectName = callButton->objectName();
+    QString videoObjectName = videoButton->objectName();
+    QString volObjectName = volButton->objectName();
+    QString micObjectName = micButton->objectName();
     loadHistoryAction->setText(tr("Load chat history..."));
+
+    if (callObjectName == QStringLiteral("green"))
+        callButton->setToolTip(tr("Start audio call"));
+    else if (callObjectName == QStringLiteral("yellow"))
+        callButton->setToolTip(tr("Accept audio call"));
+    else if (callObjectName == QStringLiteral("red"))
+        callButton->setToolTip(tr("End audio call"));
+
+    if (videoObjectName == QStringLiteral("green"))
+        videoButton->setToolTip(tr("Start video call"));
+    else if (videoObjectName == QStringLiteral("yellow"))
+        videoButton->setToolTip(tr("Accept video call"));
+    else if (videoObjectName == QStringLiteral("red"))
+        videoButton->setToolTip(tr("End video call"));
+
```

Comment:
Don't you think, that moving the two if-blocks for `callObjectName` and  `videoObjectName` to the base class is the better approach?